noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -1043,6 +1043,10 @@ pub global PROOF_TYPE_ROLLUP_HONK: u32 = 5;
 pub global PROOF_TYPE_ROOT_ROLLUP_HONK: u32 = 6;
 
 pub global TWO_POW_64: Field = 18446744073709551616;
+pub global AVM_WRITTEN_PUBLIC_DATA_SLOTS_TREE_HEIGHT: u32 = 6;
```

Comment:
can you leave a note that this is log2(MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_CALL + 1) in case we need to change it (not sure if log2 is something noir has).